docs/modules/ROOT/pages/spring-cloud-commons/common-abstractions.adoc

```diff
@@ -167,13 +167,140 @@ CAUTION: A `RestTemplate` bean is no longer created through auto-configuration.
 Individual applications must create it.
 
 The URI needs to use a virtual host name (that is, a service name, not a host name).
-The BlockingLoadBalancerClient is used to create a full physical address.
+The `BlockingLoadBalancerClient` is used to create a full physical address.
 
-IMPORTANT: To use a load-balanced `RestTemplate`, you need to have a load-balancer implementation in your classpath.
+IMPORTANT: To use a load-balanced `RestTemplate`, you need to have a Spring Cloud LoadBalancer implementation in your classpath.
 Add xref:spring-cloud-commons/loadbalancer.adoc#spring-cloud-loadbalancer-starter[Spring Cloud LoadBalancer starter] to your project in order to use it.
 
+[[multiple-resttemplate-objects]]
+=== Multiple `RestTemplate` Objects
```

Comment:
Remove the ticks around `RestTemplate`. As I mentioned in the previous comment, we have trouble with rendering the code font in headings.